tools/submission/submission-checker.py

```diff
@@ -0,0 +1,554 @@
+"""
+A checker for mlperf inference submissions
+"""
+
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import argparse
+import json
+import logging
+import os
+import re
+import sys
+
+# pylint: disable=missing-docstring
+
+
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger("main")
+
+
+MODEL_CONFIG = {
+    "v0.5": {
+        "models": ["ssd-small", "ssd-large", "mobilenet", "resnet", "gnmt"],
+        "required-scenarios-datacenter": {
+            # anything goes
+        },
+        "required-scenarios-edge": {
+            # anything goes
+        },
+        "accuracy-target": {
+            "mobilenet": ("acc", 71.68 * 0.98),
+            "resnet": ("acc", 76.46 * 0.99),
+            "ssd-small": ("mAP", 22 * 0.99),
+            "ssd-large": ("mAP", 20 * 0.99),
+            "gnmt": ("bleu", 23.9 * 0.99),
+        },
+        "performance-sample-count": {
+            "mobilenet": 1024,
+            "resnet": 1024,
+            "ssd-small": 256,
+            "ssd-large": 64,
+            "gnmt": 3903900,
+        },
+        "seeds": {
+            "qsl_rng_seed": 3133965575612453542,
+            "sample_index_rng_seed": 665484352860916858,
+            "schedule_rng_seed": 3622009729038561421,
+        },
+    },
+    "v0.7": {
+        "models": ["ssd-large", "resnet", "rnnt", "3d-unet", "dlrm", "bert"],
```

Comment:
I don't know who did this, but some model names were changed right after v0.5....
`resnet` -> `resnet50`
`ssd-large` -> `ssd-resnet34`
`ssd-small` -> `ssd-mobilenet`

Reply:
I think we always had this mixed up. So the code has the MODEL_MAPPING dict that is used to map those alternative names to the standard one. For closed division I force the official name for now but also could go via the mapping table.